guides/common/modules/proc_setting-permissions-for-the-bootstrap-script.adoc

```diff
@@ -36,23 +36,23 @@ This example creates a role with the name _Bootstrap_:
 +
 [options="nowrap", subs="+quotes,verbatim,attributes"]
 ----
-# ROLE='Bootstrap'
-hammer role create --name "$ROLE"
-hammer filter create --role "$ROLE" --permissions view_organizations
-hammer filter create --role "$ROLE" --permissions view_locations
-hammer filter create --role "$ROLE" --permissions view_domains
-hammer filter create --role "$ROLE" --permissions view_hostgroups
-hammer filter create --role "$ROLE" --permissions view_hosts
-hammer filter create --role "$ROLE" --permissions view_architectures
-hammer filter create --role "$ROLE" --permissions view_ptables
-hammer filter create --role "$ROLE" --permissions view_operatingsystems
-hammer filter create --role "$ROLE" --permissions create_hosts
+$ ROLE='Bootstrap'
```

Comment:
Are these lines supposed to be separate commands?

Reply:
To my understanding, yes. There are no trailing backslashes.